lib/livebook/intellisense/identifier_matcher.ex

```diff
@@ -48,7 +48,7 @@ defmodule Livebook.Intellisense.IdentifierMatcher do
           list(identifier_item())
   def completion_identifiers(hint, binding, env) do
     context = Code.Fragment.cursor_context(hint)
-    ctx = %{binding: binding, env: env, matcher: @prefix_matcher}
+    ctx = %{binding: binding, env: env, matcher: @prefix_matcher, hint: hint}
```

Comment:
We build similar context in `locate_identifier`, please add the hint there for completeness :)

Reply:
Thanks @jonatanklosko! I caught this and made this a draft as I should do a bigger refactor.

VSCode doesn't do anything when I mouse over a struct field, and I will follow that pattern only using them in autocomplete.

Reply:
It's probably fine to show it actually and I probably already works since `Code.Fragment.surround_context` handles it, right? Maybe showing the field itself is not particularly useful, but if we show the default value in docs it could, @josevalim?

Reply:
`surround_context` doesn't handle struct fields. At the moment, I don't think handling it is very interesting either, all we can show is the default. My suggestion is to not add/worry about this for now.

Reply:
Oh, you already handled it. So that's fine. :)